slidedeck/src/slides/05-redux/00-redux.pug

```diff
@@ -88,21 +98,22 @@ section
   h3 Immutability
   h4 Good
   script(type="text/plain", class="language-js").
-    return Object.assign({}, state, {
-      examples: state.gents.concat(action.payload)
-    });
+    case 'ADD_EMPLOYEE':
+      return Object.assign({}, state, {
```

Comment:
Just a note, that GVL is going full in on ES6 syntax so they might ask about ...spread operator

Reply:
I think object-rest-spread is ES2018? Regardless, using either can be sort of dangerous since both are shallow copies, right?

Reply:
Yep, ...spread is pretty much a drop-in simpler Object.assign, with the same immutability concerns. 

I've avoided bringing in Immer or Seamless to the GVL app so far as I felt it was too opinionated, but after discussing with them we can certainly use one or the other